pySPT/preAnalysis/pBleach_noGUI.py

```diff
@@ -0,0 +1,204 @@
+"""
+@author: Johanna Rahm
+Research group Heilemann
+Institute for Physical and Theoretical Chemistry, Goethe University Frankfurt a.M.
+
+P_bleach is the probability per particle and frame to bleach [0..1]. Calc k with exp decay function a*exp(-dt*k).
```

Comment:
adjust docstring description (add your name under authors, what is the difference to pBleach.py and why was this file created)